ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -375,14 +373,15 @@ public void shouldListConnectorPlugins() {
     // Then:
     assertThat(response.isSuccessful(), is(true));
     assertThat(response.getResponse().get(0), instanceOf(ConnectorPluginsList.class));
+    // Since no plugin is added to the connector, the size of plugins list is 0.
```

Comment:
Looks like the reason there are no connector plugins is because we're only using the built-in connectors, which are excluded from the listing: https://github.com/apache/kafka/blob/d00b7875e0ad7e005b9b54d39f47269c71e95fbe/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java#L71-L80

Reply:
nit: the comment as written makes it sound like plugins are for a specific connector, when actually they represent _types_ of connectors which are available. How about
```suggestion
    // Since no plugins have been added, the size of plugins list is 0.
```
?